lib/internal/test_runner/tests_stream.js

```diff
@@ -123,14 +123,6 @@ class TestsStream extends Readable {
     });
   }
 
-  stderr(loc, message) {
```

Comment:
this was unused. actual events are emitted here (using `kEmitMessage` directly):

https://github.com/nodejs/node/blob/c7a63b074017b57e5c8a3341151696039ae49499/lib/internal/test_runner/runner.js#L367-L371